CSV.js

```diff
@@ -195,6 +196,16 @@ function getValue(item, field) {
 			}
 			value += escapeValue(notes.join(valueSeparator));
 		break;
+		case 'date':
+			if (item.date) {
+				var dateISO = ZU.strToISO(item.date);
```

Comment:
yes, the results are the same, and this doesn't really matter. It's just weird to perform operations that are simply discarded (this one isn't terribly cheap either). You don't have to fix this, I'm just pointing it out.